Jint/Native/Array/ArrayOperations.cs

```diff
@@ -68,6 +70,60 @@ public virtual JsValue[] GetAll(Types elementTypes)
 
         public abstract void DeletePropertyOrThrow(ulong index);
 
+        public IEnumerator<JsValue> GetEnumerator()
```

Comment:
If there's an overload of concrete type `public ArrayOperationsIterator GetEnumerator()` then there would be no dispatch overhead like the interfaces have, CLR will find the version by name.

Reply:
What do you suggest? To implement it in each concrete type and call `base.GetEnumerator()` ?

Reply:
I was referring to concrete enumerator method without interface, as seen here:

https://github.com/sebastienros/jint/blob/1d29ae72bbc87406284ef3f00d7d7bd6dabcc97b/Jint/Collections/DictionarySlim.cs#L221-L235

Explicit interface implementations for generic use case and then one of concrete type to avoid boxing (struct via interface).

Reply:
Updated but this new member is not called, and we need to user OrderBy for a stable sort.